scalafmt-tests/src/test/resources/scala3/GivenPatterns.stat

```diff
@@ -0,0 +1,27 @@
+#
+<<< for comprehension
+for given Context <- applicationContexts yield ()
+>>>
+for given Context <- applicationContexts yield ()
+<<< for comprehension with parens
+for (given Context <- applicationContexts)
+yield ()
+>>>
+for (given Context <- applicationContexts)
+  yield ()
+<<< match case
+context match {
+  case (ctx @ given Context) => ctx
```

Comment:
can you add two more cases, with shorter max column, to force break before and after given? indent is of interest.

Reply:
do we want the given to break in such a situation? in the case of `case (ctx @ Context) => ctx` the line breaks only after the arrow.

Reply:
I thought it would break in the middle of the `x @ y` expression...